website/docs/components/dropdown/index.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Dropdown
+description: Displays a list (typically of actions or links) through a toggle and is identifiable by the chevron icon in the button.
```

Comment:
Would this be clearer do you think?
"Displays a list (typically of actions or links) revealed by a toggle button. Identifiable by the chevron icon in the button."

Reply:
Great idea! I'll change it.